### ISSUE TITLE
Animation break when Tab is Inactive

### ISSUE THREAD

**Issue by dalisoft** [tweenjs/tween.js#208]
Hi tween.js authors!
I using tween.js for years and now developing my animation engine (improved version of tween.js).
But has some issues/offers!
1) After animation started/under start go to another tab and return current tab, animation breaking .Link, http://tweenjs.github.io/tween.js/examples/03_graphs.html.
2) In Safari (iPhone 5S/5) after 3-4 frames stops, because while loop stops animation for high-cpu load.
3) Easing Strong not have.
4) Relative values uses when yoyo called (if repeat using).
5) IE5 shows error for Date.now and Performance.now. We need use new Date().getTime().
6) onUpdate.call(_object, value) slowing tween by 25%, if we using onUpdate(_object, value) tween will be faster.

I hope someone know these issue fixes, sorry for my bad english!

**Comment by sole** [comment]
Hey @dalisoft 

Thanks for enumerating all those flaws.

It's mostly only **me** with the help of some occasional contributors working on tween.js. So if things aren't happening, it's because I haven't experienced those issues, or I don't have the time to implement features without breaking it for other people, because I want to make sure the new features are backwards compatible and the code is of quality, and that takes time.

For this we have tests and we have rules so that this project doesn't grow out of control. And I can't, and I won't, explain again what the rules for contributing here are **because I already pointed them out to you [last year](https://github.com/tweenjs/tween.js/pull/136#issuecomment-52188063)**.

I don't even know if you're trolling with point 5 -- support for IE5? SERIOUSLY? It was released 15 years ago. [FIFTEEN](https://en.wikipedia.org/wiki/Internet_Explorer_5).

Some of those issues have already been filed. **You are not helping me by randomly opening issues** such as [this one](https://github.com/tweenjs/tween.js/issues/207). I repeat that **I do not have much time**, so dealing with vague feature requests, complain messages or PRs that do not adhere to the contribution rules does only one thing: steal time from actually working on tween.js and fixing the issues you're complaining about.

I really, really want to think you're trying to help, but you're actually not helping at all with the way you're behaving.

I'm going to close this issue and all of the other issues you have opened, because I can't cope with so many randomly unfinished requests. Feel free to open a new issue if you plan to adhere to the contributing guidelines, else I will have to block you.

Thanks.
